src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -506,16 +507,15 @@ const WorkflowView = _.flow(
 
   describeSelectionModel() {
     const { modifiedConfig: { rootEntityType }, entityMetadata, entitySelectionModel: { newSetName, selectedEntities, type } } = this.state
-    const { name } = selectedEntities
     const count = _.size(selectedEntities)
     const newSetMessage = count > 1 ? `(will create a new set named "${newSetName}")` : ''
```

Comment:
```suggestion
     const newSetMessage = (type === EntitySelectionType.processAll || count > 1) ? `(will create a new set named "${newSetName}")` : ''
```
Otherwise `newSetName` will be empty in the process-all case.